src/main/java/com/github/dockerjava/netty/NettyDockerCmdExecFactory.java

```diff
@@ -580,6 +600,24 @@ public void close() throws IOException {
         eventLoopGroup.shutdownGracefully();
     }
 
+    /**
+     * Configure connection timeout in milliseconds
+     */
+    public NettyDockerCmdExecFactory withConnectTimeout(Integer connectTimeout) {
```

Comment:
Because netty impl is buggy i keep NETTY/JERSEY choose. Connect timeout seems reasonable for any impl. But i can configure it in conditionals on my side of course.